### PR TITLE
Add the `hemlock_house` encounter set

### PR DESCRIPTION
**Pull request by fspoettel**
This introduces a new `type` - `enemy_location`. The corresponding arkhamdb PR to support it is linked below.

ArkhamDB PR: https://github.com/Kamalisk/arkhamdb/pull/679

### CODE REVIEW

File: pack/fhv/fhvc.json [hunk 1]
```diff
@@ -60,18 +60,742 @@
...
+    "back_link": "10524b",
+    "code": "10524",
+    "encounter_code": "hemlock_house",
+    "encounter_position": 2,
+    "faction_code": "mythos",
+    "name": "The Predatory House",
+    "pack_code": "fhvc",
+    "position": 24,
+    "quantity": 1,
+    "text": "Create a separate bag consisting of the following tokens: [cultist], [tablet] and [elder_thing]. (If you do not have a second bag, you can use a cup, bowl, or other opaque container.) This is the predation bag, and these are predation tokens. They are not chaos tokens.\nThen, add the following tokens to the predation bag based on the current day:\n- <b>Day 1:</b> Add [cultist], [cultist] to the predation bag.\n- <b>Day 2:</b> Add [cultist] to the predation bag.\n- <b>Day 3:</b> No change is made.\nSome abilities will instruct investigators to make a predation test. A predation test may cause the house to come alive and prey on the investigators. To resolve a predation test, reveal a random token from the predation bag and resolve the effects for that token as indicated on the other side of this card.\nFlip this card over and place it next to the scenario reference card.",
```
Comment (drawntotheflame) [inline pack/fhv/fhvc.json]:
```suggestion
    "text": "Create a separate bag consisting of the following tokens: [cultist], [tablet], and [elder_thing]. (If you do not have a second bag, you can use a cup, bowl, or other opaque container.) This is the predation bag, and these are predation tokens. They are not chaos tokens.\nThen, add the following tokens to the predation bag based on the current day:\n- <b>Day 1:</b> Add [cultist], [cultist] to the predation bag.\n- <b>Day 2:</b> Add [cultist] to the predation bag.\n- <b>Day 3:</b> No change is made.\nSome abilities will instruct investigators to make a predation test. A predation test may cause the house to come alive and prey on the investigators. To resolve a predation test, reveal a random token from the predation bag and resolve the effects for that token as indicated on the other side of this card.\nFlip this card over and place it next to the scenario reference card.",
```

File: pack/fhv/fhvc.json [hunk 2]
```diff
@@ -60,18 +60,742 @@
...
+    "type_code": "agenda"
+  },
+  {
+    "back_name": "Walls of Flesh",
+    "back_text": "<b>If there are more seals in play than enemy-locations in the victory display:</b>\n<blockquote>Your efforts to suppress the strange infestation seem to have angered it. A noisome screech shakes the foundation of the house, and the air floods with a fetid stench of something awful down below.</blockquote>\nPut the set-aside Shapeless Cellar enemy-location into play on the Cellar Floor, as close to the middle column as possible.\nAdvance to the set-aside act 2a \"The Heart of the House\" and remove act 2a \"Against the House\" from the game.\n<hr>\n<b>If there are more enemy-locations in the victory display than seals in play:</b>\n<blockquote>The foundations of the house shake as another room threatens to collapse. If you bring this house down, the abomination that lives inside it won't survive long.</blockquote>\nAdvance to the set-aside act 2a \"Against the House\" and remove act 2a \"The Heart of the House\" from the game.",
```
Comment (drawntotheflame) [inline pack/fhv/fhvc.json]:
```suggestion
    "back_text": "<b>If there are more seals in play than enemy-locations in the victory display:</b>\n<blockquote><i>Your efforts to suppress the strange infestation seem to have angered it. A noisome screech shakes the foundation of the house, and the air floods with a fetid stench of something awful down below.</i></blockquote>\nPut the set-aside Shapeless Cellar enemy-location into play on the Cellar Floor, as close to the middle column as possible.\nAdvance to the set-aside act 2a \"The Heart of the House\" and remove act 2a \"Against the House\" from the game.\n<hr>\n<b>If there are more enemy-locations in the victory display than seals in play:</b>\n<blockquote><i>The foundations of the house shake as another room threatens to collapse. If you bring this house down, the abomination that lives inside it won't survive long.</i></blockquote>\nAdvance to the set-aside act 2a \"Against the House\" and remove act 2a \"The Heart of the House\" from the game.",
```

File: pack/fhv/fhvc.json [hunk 3]
```diff
@@ -60,18 +60,742 @@
...
+    "encounter_code": "hemlock_house",
+    "encounter_position": 7,
+    "faction_code": "mythos",
+    "illustrator": "Romain Leguay",
+    "name": "Strange Infestation",
+    "pack_code": "fhvc",
+    "position": 29,
+    "quantity": 1,
+    "stage": 1,
+    "text": "Clues cannot be discovered from locations with no investigators.\n[action] If your location is unsealed and [[Dormant]], investigators at that location spend 1 [per_investigator] clues, as a group: Place 1 resource on it, as a seal.\n<b>Objective</b> - At the end of the round, if a total of 7 locations are sealed and / or in the victory display, advance.",
```
Comment (drawntotheflame) [inline pack/fhv/fhvc.json]:
```suggestion
    "text": "Clues cannot be discovered from locations with no investigators.\n[action] If your location is unsealed and [[Dormant]], investigators at that location spend 1 [per_investigator] clues, as a group: Place 1 resource on it, as a seal.\n<b>Objective</b> - At the end of the round, if a total of 7 locations are sealed and/or in the victory display, advance.",
```

File: pack/fhv/fhvc.json [hunk 4]
```diff
@@ -60,18 +60,742 @@
...
+    "type_code": "act"
+  },
+  {
+    "encounter_code": "hemlock_house",
```
Comment (drawntotheflame) [inline pack/fhv/fhvc.json]: Please alphabetise this card entry.

File: pack/fhv/fhvc.json [hunk 5]
```diff
@@ -60,18 +60,742 @@
...
+    "flavor": "Sealing this creature away won't solve anything. Whatever sort of monster has taken over Hemlock House, it needs to be destroyed.",
+    "illustrator": "Romain Leguay",
+    "name": "Against the House",
+    "quantity": 1,
+    "stage": 2,
+    "text": "Clues cannot be discovered from locations with no investigators.\n<b>Objective</b> - Bring this house down! If no locations are in play, immediately advance <i>(this occurs before investigators would be defeated by not being at a location)</i>.",
+    "type_code": "act"
+  },
+  {
+    "encounter_code": "hemlock_house",
```
Comment (drawntotheflame) [inline pack/fhv/fhvc.json]: NB alphabetise please.

File: pack/fhv/fhvc.json [hunk 6]
```diff
@@ -60,18 +60,742 @@
...
+    "pack_code": "fhvc",
+    "encounter_position": 9,
+    "code": "10531",
+    "position": 31,
+    "back_flavor": "The house shudders, then stills. Sealing off the crevices and nooks where the creature was breaching through the house's walls seems to have forced it to retreat, or perhaps lay dormant for a while. Hopefully it won't resurface, but in the meantime, the legacy of the Hemlock family still stands. Maybe in time, they can find a way to exterminate the creature for good.",
+    "back_name": "Laid to Rest",
+    "back_text": "Flip each enemy-location to its location side. Place 1 resource, as a seal, on each unsealed [[Dormant]] location.\n<b>(→R1)</b>. <i>(page 18)</i>.",
```
Comment (drawntotheflame) [inline pack/fhv/fhvc.json]:
```suggestion
    "back_text": "Flip each enemy-location to its location side. Place 1 resource, as a seal, on each unsealed [[Dormant]] location.\n<b>(→R1)</b> <i>(page 18)</i>.",
```

File: pack/fhv/fhvc.json [hunk 7]
```diff
@@ -60,18 +60,742 @@
...
+    "flavor": "If you want to seal the house for good, you must delve into the loathsome sepulchre below.",
+    "illustrator": "Romain Leguay",
+    "name": "The Heart of the House",
+    "quantity": 1,
+    "stage": 2,
+    "text": "Clues cannot be discovered from locations with no investigators.\n[action] If your location is unsealed and [[Dormant]], investigators at that location spend 1 [per_investigator] clues, as a group: Place 1 resource on it, as a seal.\n<b>Forced</b> - After an enemy-location attacks you: Move (one location at a time) to Shapeless Cellar.\n<b>Objective</b> - When the Shapeless Cellar is in the victory display, advance.",
+    "type_code": "act"
+  },
+  {
+    "back_link": "10532b",
```
Comment (drawntotheflame) [inline pack/fhv/fhvc.json:226]: NB alphabetise please.

File: pack/fhv/fhvc.json [hunk 8]
```diff
@@ -60,18 +60,742 @@
...
+    "position": 32,
+    "quantity": 1,
+    "shroud": 3,
+    "text": "Massive. Cannot make attacks of opportunity. Living Bedroom gets +2 [per_investigator] health.\n<b>Forced</b> - When this enemy-location is revealed: Place 1 doom on it.",
+    "traits": "Room. Monster. Elite.",
+    "type_code": "enemy_location",
+    "victory": 0
+  },
+  {
+    "back_link": "10533b",
```
Comment (drawntotheflame) [inline pack/fhv/fhvc.json:268]: NB alphabetise please.

File: pack/fhv/fhvc.json [hunk 9]
```diff
@@ -60,18 +60,742 @@
...
+    "position": 33,
+    "quantity": 1,
+    "shroud": 3,
+    "text": "Massive. Cannot make attacks of opportunity. Living Bedroom gets +2 [per_investigator] health.\n<b>Forced</b> - When this enemy-location is revealed: It attacks each investigator at this location.",
+    "traits": "Room. Monster. Elite.",
+    "type_code": "enemy_location",
+    "victory": 0
+  },
+  {
+    "back_link": "10534b",
```
Comment (drawntotheflame) [inline pack/fhv/fhvc.json:311]: NB alphabetise please.

File: pack/fhv/fhvc.json [hunk 10]
```diff
@@ -60,18 +60,742 @@
...
+    "position": 34,
+    "quantity": 1,
+    "shroud": 3,
+    "text": "Massive. Cannot make attacks of opportunity. Living Bedroom gets +2 [per_investigator] health.\n<b>Forced</b> - When this enemy-location is revealed: Make a predation test.",
+    "traits": "Room. Monster. Elite.",
+    "type_code": "enemy_location",
+    "victory": 0
+  },
+  {
+    "back_link": "10535b",
```
Comment (drawntotheflame) [inline pack/fhv/fhvc.json:354]: Nb alphabetise please.

File: pack/fhv/fhvc.json [hunk 11]
```diff
@@ -60,18 +60,742 @@
...
+    "position": 36,
+    "quantity": 1,
+    "shroud": 4,
+    "text": "Massive. Cannot make attacks of opportunity.\n<b>Forced</b> - When this enemy-location is revealed, if there are 1 or more clues on it: It attacks each investigator at this location.",
+    "traits": "Room. Monster. Elite.",
+    "type_code": "enemy_location",
+    "victory": 0
+  },
+  {
+    "encounter_code": "hemlock_house",
```
Comment (drawntotheflame) [inline pack/fhv/fhvc.json]: NB alphabetise please.

File: pack/fhv/fhvc.json [hunk 12]
```diff
@@ -60,18 +60,742 @@
...
+    "enemy_evade": 2,
+    "enemy_fight": 4,
+    "enemy_horror": 3,
+    "faction_code": "mythos",
+    "health": 4,
+    "hidden": true,
+    "illustrator": "Pixoloid Studios",
+    "name": "Living Library",
+    "pack_code": "fhvc",
+    "position": 39,
```
Comment (drawntotheflame) [inline pack/fhv/fhvc.json]:
```suggestion
    "position": 40,
```

File: pack/fhv/fhvc.json [hunk 13]
```diff
@@ -60,18 +60,742 @@
...
+    "encounter_position": 22,
+    "enemy_damage": 1,
+    "enemy_evade": 2,
+    "enemy_fight": 4,
+    "enemy_horror": 1,
+    "faction_code": "mythos",
+    "flavor": "\"It's got me!\"",
+    "health": 2,
+    "illustrator": "JB Casacop",
+    "is_unique": true,
```
Comment (drawntotheflame) [inline pack/fhv/fhvc.json]:
```suggestion
  
```

Comment (drawntotheflame) [reply pack/fhv/fhvc.json]: Please delete line.

File: pack/fhv/fhvc.json [hunk 14]
```diff
@@ -60,18 +60,742 @@
...
+    "name": "Grappling Spawn",
+    "pack_code": "fhvc",
+    "position": 44,
+    "quantity": 2,
+    "text": "Hunter. <b>(Night [night])</b> Retaliate.\nGrappling Spawn gets +X health, where X is its current floor (to a minimum of +1 health).",
+    "traits": "Monster. Mutated.",
+    "type_code": "enemy"
+  },
+  {
+    "encounter_code": "hemlock_house",
```
Comment (drawntotheflame) [inline pack/fhv/fhvc.json:750]: NB please alphabetise.

File: pack/fhv/fhvc.json [hunk 15]
```diff
@@ -60,18 +60,742 @@
...
+    "faction_code": "mythos",
+    "illustrator": "Darek Zabrocki",
+    "name": "Out of the Walls",
+    "quantity": 4,
+    "text": "<b>Revelation</b> - Test [agility] (3). If you fail and...\n- ...your location is an enemy, take 1 damage and 1 horror.\n- ...your location is unsealed and [[Dormant]], flip it over.\n- ...your location is sealed, discard 1 card at random from your hand.",
+    "traits": "Hazard.",
+    "type_code": "treachery"
+  },
+  {
+    "encounter_code": "hemlock_house",
```
Comment (drawntotheflame) [inline pack/fhv/fhvc.json:764]: NB alphabetise please.

File: pack/fhv/fhvc.json [hunk 16]
```diff
@@ -60,18 +60,742 @@
...
+    "faction_code": "mythos",
+    "illustrator": "Jeffrey Lai",
+    "name": "Pulled In",
+    "quantity": 2,
+    "text": "<b>Revelation</b> - If there are no enemy-locations in play, Pulled In gains surge. Otherwise, test [combat] (4). If you fail, choose the nearest enemy-location and move (one location at a time) to it. If you failed by 4 or more, the chosen enemy-location attacks you.",
+    "traits": "Blunder.",
+    "type_code": "treachery"
+  },
+  {
+    "encounter_code": "hemlock_house",
```
Comment (drawntotheflame) [inline pack/fhv/fhvc.json:780]: NB alphabetise please.

File: pack/fhv/fhvc.json [hunk 17]
```diff
@@ -60,18 +60,742 @@
...
+    "encounter_position": 30,
+    "code": "10547",
+    "position": 47,
+    "clues": 3,
+    "enemy_damage": 1,
+    "enemy_evade": 2,
+    "enemy_fight": 3,
+    "enemy_horror": 2,
+    "faction_code": "mythos",
+    "health": 5,
```
Comment (drawntotheflame) [inline pack/fhv/fhvc.json:788]:
```suggestion
    "health_per_investigator": true,
    "health": 5,
```

**Review by drawntotheflame — COMMENTED** [review]
Great job. Only a few wee tweaks, and a nudge to alphabetise some entries! Thanks very much.

**Comment by fspoettel** [comment]
thanks @drawntotheflame, feedback implemented.

**Comment by fspoettel** [comment]
@zzorba can this one be merged? thank you 🙏

**Pull request merged**